packages/ui-test-queries/src/utils/events.ts

```diff
@@ -347,8 +355,14 @@ function fireEvent(element, event) {
 Object.entries(eventMap).forEach(([key, { EventType, defaultInit }]) => {
   const eventName = key.toLowerCase()
 
-  fireEvent[key] = (node, init = {}) => {
+  // not nice, but this is a very weird code :/
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  fireEvent[key] = (node: Element, init: Record<string, unknown> = {}) => {
     const eventInit = { ...defaultInit, ...init }
```

Comment:
Again, some really weird code (adding additional function properties to a function) that I could not type properly without refactoring